apps/rewards/src/attestation/attestation.entity.ts

```diff
@@ -0,0 +1,31 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm'
+
+export enum RewardStatus {
+  Created = 'Created',
+  Sent = 'Sent',
+  Failed = 'Failed'
+}
+
+@Entity()
+export class Attestation {
+  public static of(params: Partial<Attestation>): Attestation {
+    const attestation = new Attestation()
+    Object.assign(attestation, params)
+    return attestation
+  }
+
+  @PrimaryGeneratedColumn('uuid')
+  id: string
+
+  @Column({ unique: true })
+  txHash: string
```

Comment:
In the case that we batch submit attestations in a single transaction in the future, then multiple attestations would map to the same attestation hash. Given that Valora isn't the only wallet that uses AS protocol, I don't think we can guarantee this won't be the case.

Reply:
We could add an `issuer` column and do the unique constraint in the [txHash, issuer] pair